Testscripts/Linux/gpu-driver-install.sh

```diff
@@ -29,45 +28,55 @@ grid_driver="https://go.microsoft.com/fwlink/?linkid=874272"
 
 #######################################################################
 function InstallCUDADrivers() {
+    LogMsg "Starting CUDA driver installation"
     case $DISTRO in
     redhat_7|centos_7)
-        CUDA_REPO_PKG="cuda-repo-rhel7-${CUDADriverVersion}.x86_64.rpm"
-        LogMsg "Using ${CUDA_REPO_PKG}"
+        CUDA_REPO_PKG="cuda-repo-rhel7-$CUDADriverVersion.x86_64.rpm"
+        LogMsg "Using $CUDA_REPO_PKG"
 
         wget http://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/"${CUDA_REPO_PKG}" -O /tmp/"${CUDA_REPO_PKG}"
         if [ $? -ne 0 ]; then
-            LogErr "Failed to download ${CUDA_REPO_PKG}"
+            LogErr "Failed to download $CUDA_REPO_PKG"
             SetTestStateAborted
             return 1
+        else
+            LogMsg "Successfully downloaded the $CUDA_REPO_PKG file in /tmp directory"
         fi
 
-        rpm -ivh /tmp/"${CUDA_REPO_PKG}"
+        rpm -ivh /tmp/"$CUDA_REPO_PKG"
+        LogMsg "Installed the rpm package, $CUDA_REPO_PKG"
```

Comment:
Should say "Installing the rpm package..." or "Starting installation of the rpm package".

Reply:
The Logging is after the command execution. So 'past' statement is correct.